### PR TITLE
[Tests/Filter/Reload] Check existence of TFLite filters before start

### PR DESCRIPTION
**Pull request by wooksong**
The test cases for reload models require the tensor filter extension for Tensorflow Lite. As other test cases do, this patch adds a procedure that checks existence of the tensorflow-lite filters before starting the test.

Signed-off-by: Wook Song <wook16.song@samsung.com>
See also: #2164 

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

File: tests/nnstreamer_filter_reload/runTest.sh [hunk 1]
```diff
@@ -19,6 +19,51 @@ testInit $1
 
 # NNStreamer and plugins path for test
 PATH_TO_PLUGIN="../../build"
+# This test is valid only with the tensor filter extension for tflite
+if [[ -d $PATH_TO_PLUGIN ]]; then
+    ini_path="${PATH_TO_PLUGIN}/ext/nnstreamer/tensor_filter"
+    if [[ -d ${ini_path} ]]; then
+        check=$(ls ${ini_path} | grep tensorflow-lite.so)
```
Comment (jaeyun-jung) [inline tests/nnstreamer_filter_reload/runTest.sh]: better to set SO_EXT as you fixed today?

Comment (wooksong) [reply tests/nnstreamer_filter_reload/runTest.sh]: Ah:( my bad...

File: tests/nnstreamer_filter_reload/runTest.sh [hunk 2]
```diff
@@ -14,11 +14,60 @@ if [[ "$SSATAPILOADED" != "1" ]]; then
...
+if [ -z ${SO_EXT} ]; then
+    SO_EXT="so"
+fi
+
 # This is compatible with SSAT (https://github.com/myungjoo/SSAT)
 testInit $1
 
 # NNStreamer and plugins path for test
 PATH_TO_PLUGIN="../../build"
+# This test is valid only with the tensor filter extension for tflite
```
Comment (jijoongmoon) [inline tests/nnstreamer_filter_reload/runTest.sh]: It looks much better for me. Sure. we have not to use relative directory path.

File: tests/nnstreamer_filter_reload/runTest.sh [hunk 3]
```diff
@@ -14,11 +14,60 @@ if [[ "$SSATAPILOADED" != "1" ]]; then
...
+if [[ -d $PATH_TO_PLUGIN ]]; then
+    ini_path="${PATH_TO_PLUGIN}/ext/nnstreamer/tensor_filter"
+    if [[ -d ${ini_path} ]]; then
```
Comment (jijoongmoon) [inline tests/nnstreamer_filter_reload/runTest.sh]: It would be better to make the path clear!

**Pull request merged**